src/Symfony/Component/Ldap/Security/CheckLdapCredentialsListener.php

```diff
@@ -70,9 +69,6 @@ public function onCheckPassport(CheckPassportEvent $event)
         }
 
         $user = $passport->getUser();
-        if (!$user instanceof PasswordAuthenticatedUserInterface) {
-            trigger_deprecation('symfony/ldap', '5.3', 'Not implementing the "%s" interface in class "%s" while using password-based authenticators is deprecated.', PasswordAuthenticatedUserInterface::class, get_debug_type($user));
```

Comment:
Should this now also a exception?

Reply:
IMHO it does not need to, we can just remove the deprecation here. 
Reason for this notice to be here is that in order to remove `getPassword()` from the generic UserInterface, we needed to trigger deprecation notices from as much places as possible to make sure developers get it. Here we don't use anything from `PasswordAuthenticatedUserInterface` so nothing to do.